UI/src/view/navigationpanel/massTransitStopBox.js

```diff
@@ -94,9 +118,10 @@
     };
 
     this.toolSelection = new me.ToolSelection([
-      new me.Tool('Select', me.selectToolIcon, selectedMassTransitStop),
-      new me.Tool('Add', setTitleTool(me.addToolIcon, 'Lisää pysäkki'), selectedMassTransitStop),
-      new me.Tool('AddTerminal', setTitleTool(me.terminalToolIcon, 'Lisää terminaalipysäkki'), selectedMassTransitStop)
+      new me.Tool('Select', me.selectToolIcon, selectedMassTransitStop ),
+      new me.Tool('Add', setTitleTool(me.addToolIcon, 'Lisää pysäkki'), selectedMassTransitStop ),
+      new me.Tool('AddTerminal', setTitleTool(me.terminalToolIcon, 'Lisää terminaalipysäkki'), selectedMassTransitStop),
+      new me.Tool('AddPointAsset', setTitleTool(me.pointAssetToolIcon, 'Add point asset'), selectedMassTransitStop )
```

Comment:
Change 'Add point asset' to something more descriptive. 'Lisää palvelupiste' could be an option

Reply:
Done